javascript/ql/src/experimental/Security/CWE-326/ecb_rejection.ql

```diff
@@ -0,0 +1,33 @@
+/**
+ * @name ECBRejection
```

Comment:
```suggestion
 * @name Use of weak block mode
```

I don't really like the name `ECBRejection`. 
What does `Rejection` mean here? (maybe you meant `Injection`).  
But in both cases this is the first use ever of that word, and I prefer if we don't invent a new word.  

The query finds uses of weak block modes (where `ECB` happens to be the only one recognized as weak in the implementation).   
So I think a name like `Use of weak block mode` fits better.